cdc/sink/codec/json.go

```diff
@@ -102,6 +102,15 @@ func (c *column) decodeCanalJSONColumn(name string, javaType JavaSQLType) *model
 		log.Panic("canal-json encoded message should have type in `string`")
 	}
 
+	if javaType == JavaSQLTypeBIT {
```

Comment:
Is this a bugfix?

Reply:
no, this is related to decoder logic.

consider the `bit(64)` with value `0b'100001`, this is interpreted as int `65`. When the decoder handle string `"65"`, it was incorrect data. in this case, `int 65` is not identical to `string "65"` .